src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -73,7 +73,7 @@ internal static DerSequenceReader CreateForPayload(byte[] payload)
         {
             return new DerSequenceReader(true, payload);
         }
-
+        
```

Comment:
Particularly since the only edit to this file is to add 8 spaces, please purge the spaces and squash.